.travis.yml

```diff
@@ -12,10 +12,16 @@ branches:
   only:
     - master
 
+before_script:
+  - "export DISPLAY=:99.0"
+  - "sh -e /etc/init.d/xvfb start"
+  - sleep 3 # give xvfb some time to start
+  - npm run webdriver-manager-start &> /dev/null &
```

Comment:
Strange, you would think so but it was working without it but I'll add it for sake of clarity